src/kernels/errors/invalidRemoteJupyterServerUriHandleError.ts

```diff
@@ -15,10 +15,8 @@ import { BaseError } from '../../platform/errors/types';
  */
 export class InvalidRemoteJupyterServerUriHandleError extends BaseError {
     constructor(
-        public readonly providerId: string,
-        public readonly handle: string,
-        public readonly extensionId: string,
-        public readonly serverId: string
+        public readonly serverHandle: { extensionId: string; id: string; handle: string },
```

Comment:
Handle Id and Handle are not unique, two extensions could have the same two values.